src/config/default.json

```diff
@@ -10,6 +10,13 @@
 			"http://api-01.us-west-1.testnet.symboldev.network:3000"
 		]
 	},
+	"nis1Nodes": [
```

Comment:
Any reason we need this?

Reply:
Changed solution a bit. Node list removed